docs/AboutUs.adoc

```diff
@@ -4,53 +4,53 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 3 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
-{empty} +
+ORGANice Transplant Manager is being developed by team https://github.com/AY1920S1-CS2103T-T13-1[CS2103T-T13-1]. +
+
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
```

Comment:
Perhaps we can mention that we are a student team, and put in our team id also? This description looks like we are actually a developer team lmao

Reply:
maybe like `We are a group of students in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore]`

Reply:
Sounds good